### PR TITLE
Improve the G in Gutenberg

### PR DESCRIPTION
**Pull request by pento**
## Description

While the "Welcome to Gutenberg" message at the end of `bin/setup-local-env.sh` is cute, it was missing something. Something important. Something magical. Something... wavy.

This PR improves the G, to more closely match Gutenberg's actual logo.

## Screenshots

*Before*

The Gutenberg ASCII art at the end of the setup script

*After*

The Gutenberg ASCII art at the end of the setup script

### CODE REVIEW

**Comment by jdevalk** [comment]
https://www.youtube.com/watch?v=Xi3aEGo8y-E

**Comment by pento** [comment]
You have distracted from my creative process

**Review by Soean — APPROVED** [review]

**Review by youknowriad — CHANGES_REQUESTED** [review]
I should have made the PR 😆

**Comment by pento** [comment]
@youknowriad:

https://twitter.com/kanyewest/status/987727961477472257

@jasmussen:

https://twitter.com/kanyewest/status/988571191651713024

File: bin/setup-local-env.sh [hunk 1]
```diff
@@ -16,8 +16,8 @@ cd "$(dirname "$0")/.."
 . "$(dirname "$0")/install-docker.sh"
 
 ! read -d '' GUTENBERG <<"EOT"
-,⁻⁻⁻.       .                 |
-|  _. .   . |---  ,---. ,---. |---. ,---. ,---. ,---.
+,⁻⁻⁻·       .                 |
+|  ،~’.   . |---  ,---. ,---. |---. ,---. ,---. ,---.
```
Comment (aduth) [inline bin/setup-local-env.sh]: The standard tilde is perhaps a bit too harsh. What do you think of "Swung Dash" `⁓` (U+2053)?

**Comment by jasmussen** [comment]
"Build Tooling"

**Comment by mzorz** [comment]
I love it.  It brings memories of my good old BBS times.

**Comment by notnownikki** [comment]
Retina displays will need this alternate version

```
`shhhhhdh-.yhhhhhhhhhhhho  .od+          .-//:-`    `.://:-`         .yh/` `+hhhhhhhhhhhhy-+hhhhhhh+
.mm/:::/:-::////+ss+////- /mm/-+s+.   `+dmdhyhmmh++sdmdhydmmy-    -+s+:sNh. .::/+ss/:::::/--:::::oNh
.dh`    `oy`   .hMNmmy:` .dN.-mNsmm.`/mmo.    `oNMMMd-`   `:hNy. :mmsNd.sM+ `:ymmNMh.    so.     :my
 ..  `:sy:`/ds. .sNdohmds+mN-.ymdmy.sNd//+syhddNmssyNmddhyo+:oNd-.ymdms.hMoodNhohNy. .sd+``+yo:` `-.
.dy`-dmdMs``oNNs:`-ohdNMNmdmm+:://+dMMmdhso/:-hN+  .dN/:/+oyhdNMNs///:/dNdmNMNdhs-`-smNs. .dMhmh`:my
-Nh`oMmmd:   -dMNdo:-.--..`.omNmmmdy+:.``    -Nd.   oNs     `.-/ohdmmNNs-`..-..-:odNMd-    +mdmM-:Nh
-Nh`.oo:.     .hMMMmddysoosydmho:.`    `-:/oshNs    -NNyo+/-.`   ``:ohmdhsoosyhdmMMMd.      ./o+`:Nh
-Nh` `+hhdh+.  -dMNms//+oo+/-.`  `.:+yhddhysosNy.  `/Ndooyyhddhs/-`  `.-/+oo+//smNMm-  .+hdhh+`  :Nh
-Nh` `/+::+mm:  oNyomd/`      `-ohddyo:-```  `omdyshmh-   ``.-/ohddy/.      `:hmosNs  :mm+::+/`  :Nh
-Nh`       /My  /Nh`-yNh:::-:sddy+-`      ``..-/omNh+:-.``      `.:ohmh+:///ymy- yN+  yM/        :Nh
-Nh`     `-hN+  oNs  `+NNhhmNd/.`    `.:+shhddhdNMMNmhhdhhyo/-.     `-sNmhymMs`  +Ns` +Nh-`      :Nh
-Nd+syyhhhdy:` /mm.   +My``oNy`  `./shdhyo/-.-ommo/hNy/..:+oyddho:`   /Nh.`/Mh`  `dN/ `:ydhhhyys/+Nh
-NNds++/:-.` `omd:   .sNmysdNs`./ydds/-`   .+dmo.  `:hmy-   `.-+yddo  :NNhydNd:   -hmo` `.-:/++sdNMh
yNs./hddy:  -hNo.  `:dmo/o+/yNhmdo:`     `/dmo.      `/hms-      .//-smh/://:yNs.  `oNh-  .sddho.sMd
Nm-`mm+oMy`:dm/   `oNh-/hy--yNmNo.     `+dmo.          `:dms-     `+md+-:`:dy:+md:   /md:`oMy/dN--mN
hNo.+hddh::md:   .hNo-sNh:omd/.omd/`   +ds-              `/ys   `/hNo. +mh/+dm+:hN+`  :dN:.sdddo.oNh
.smho////omd:   -dN+-hmo:dmo`   -ymh-   `                  `   .ymy-    :hNo-yNs-yNo`  -dmo///+ohms.
 `-+syhhdNN:   .hN/-dm//mm:      `:hy.                        `sh/`      `oNy-sNy.hN+`  -NMdhhys+-` 
-hs/-..`sMo   `yN+.dN//mm.     `-. ``                          `` `-.      +Nh.sNs.hN/   /Nh.`..-oho
:Mmddds:Nd`   +My`yM+-Nm-    `:ymo`              ```              -dmo.    `oNy.yM/:mm`  `sM+-ddddMh
:My````hN/   `mm:-Mh.dN/   `:hmy-            ./syhhhyo/.           `+dmo.   `yN+:md`oNo   -mN.`` :Nh
:My`  .Nd-   +Ns sN++Ny`  -yNy:            `omms/---/omm+            `+mmo.  .md-hN/-md.   sM+   :Nh
:My`  /My` `.hN/ .:-dN- -hNh:              yMy`       -dM:             `omm+. sMo./``hN+`` :Nd`  :Nh
:My`  oNyoydmmNmh+ :md-yNh:`              -Mm-         :+`  `-`          .+mm+:My``+hmNmmdyomm-  :Nh
:My`  yMNy+-.` -yNsoNNNh:                 +Mh.           `-omh.            `omNMd:hNs. `.:odMm:  :Nh
:My`  hMy.      :NNmMMy.                  +Mh.     .+yhdmmds:               `:NMNmMm-     `-mN/  :Nh
:My`  sMNmds+::omN:/Nmmmo`                +Mh.   `smy/:dM/                 -yNdNd.+Ndo:/+sdNMm:  :Nh
:My`  oNs./oyNNh+. :md.omm+.              :Md-   -y:  .hM/               -yNh::My` .+mMdyo:-mm-  :Nh
`:-   :Mh`   yN+   `hN: .omm+`            `dMs        .dM:             -yNd:` yN+   `dN/   :Nd`  `:-
-d+   `mm:   :Nh`   /Ny`  .omm+`           .yNh/-.`.-/dNs`           -yNh:`  :Nd.   :Nh`   sMo   -ys
.os+`  oMs`   dN/   `hN+    .smm+`          `-ohdhhhhyo-`          .ymd/`   .hN/   `yN/   -dN.  :ss+
.sys.  .dN-   -Nd.   .mN/     .omd+`            `````            -smh/`    `sMs    +Nh   `sMo  `/yy+
-yhhs.  /Nh`   +Nh`   :dN:      -smd/`                         .smd/`     `yNs`   :md-   :Nd.  /hhhs
-yyyyo` `sNs`  `sNs`   -dN+`      -smd/`                     -smd/`      -yNo`   -mm:   .mm:  /yyhyo
.ossso:` `yNo`  `oNy.   .yNy-       -smh/`                 .omd+`      `+mm/`   /mm:   .hN/  -+ssss+
-yyyyhy:  `hNo`   oNd-   `/dmo.       -ymh/`             .omd+.      `:hNy-   `oNd-   -hN/  `syyyyyo
-hhhhhhho` .sNs.  `/dm+.```.odms-`     `:ymh/`         .omd+.      `/hmy:````-hNy-   :mm/  :yhhhhhhs
`:oy/:+so+- `+NhosyhdNNdhhhhyymMNh+.`    `-ymh:`     .odm+.     `:odMNdyhhhhhmNmhhyosmd: `/+so/:os+-
`oNh.`+hhh+`` :dMm+:---.---:/+oyhdmmho/-`  `:ymh:` .+ddo.  `.-+ydmmdhso+:----.---:yNNs` `-yhhs-`/md-
-Nm..shhhhhho` .omh/.-://+++//:---:/oyddhyo/::+dNyodNs::/+syddhs/:---:://+++//:--smh:  :yhhhhhh/`+Ny
-Nh`.o+-:ossoo+. .smNNdyysssyyhhddhyo/:-:+oyhhhdmNMNmhhhhso/:-:+syhdhhhyysssyymNNd/` /oosss+--o/ :Nd
`hN+-o: .+sssss:..`-smh+........--:+shdhs/.```...hN+..````-+yddho/:-........:smd+....osssoo: `+/.hN/
 .ymy/-````so.yddh/  .odmmNmhdddhys+:--/sdmh+-`  yN/  `-odmh+:--/oshhddhdmNmmh/`  yddh/:y:```.:odm+`
  `/mNNdhyys+-...`     `:ymmy+:-:/oyddhs:.-odmy:`hN/.+hmh/-./yddhs+/:::odmdo-`    ```..:osyhhmNMy-  
 `omho/oy+/ohmy-          `:sddho:.``./ymms-`:yNmmNdmmo-./ymdo:.``-+ydmh+-          `+mmy+/sy:+yNh: 
`sNs``.mm:  `:md-           `./smNNdyo/-:+hmh/oNh/-omd-+dms/-:+shmNNh+-`            oNy.  `yN+``:mm-
-NNoshmh:     sMo.`       .+ydmdhhdmNMMNddhmMMMd.   +NMMMdddmMMMNdhhhdmds:`      `.:dN-    .smdyohMs
:Nmy+/-.    `:mMmdmh:   .ymd+-..://::+yNd+::/oNm:  `sNy/:::smdo:-:/:-../ymd/`  .sddmNMy.    `.:/shNy
 .`   /dy+/+yNMm:.:mN. :mm/` :hmdhhmms--hNo   :mm/-yNy`  -dNo.+dmdhdmd+` .yNs``oMs..sMMds//ohy.  `.`
 -/////omMMNdoNNy+sNm+omM/` +Mh:   :hMNosmM/   .sNNd/`  `mMhodMmooo-.sNh` .dMy/yMm+omMysmMMNh+/////-
`+yyyyyyyyyyyyyyyhyyyyyyhs` :+.    :oosoooo:     oh-     +ooooooooo- `/+` /yyyyyyyyyyyyyyyyyyyyyyyy+
```

**Comment by aaronjorbin** [comment]
This PR seems like it was 
jw9hbla

**Comment by notnownikki** [comment]
I'm pretty sure this impacts embeds in some way.

**Comment by mcsf** [comment]
Can we get some regression tests for this before merging?

**Comment by pento** [comment]
@notnownikki: I love the work on the retina image! Let's do a followup PR that detects the screen DPI in the terminal, as it feels like it might be a bit out of scope for this PR.

@aaronjorbin: You have no idea, this is like my 10th revision.

@jasmussen: Thank you for the accessibility aspect, it's important to consider. The good news is, the poor contrast is entirely down to my choice of Terminal themes: most default themes will display it as #FF00FF (or something similar) against #000000 (again, or something similar), which has [pretty good contrast](https://webaim.org/resources/contrastchecker/?fcolor=FF00FF&bcolor=000000).

There are also a many high contrast Terminal themes available for every terminal app and platform that you choose to use.

Though, this does bring up the point of screen readers. Obviously, no screen reader would be able to make sense of this output, but I'm not aware of a way to provide a plain text alternative for it. Interestingly, [MUDs](https://en.wikipedia.org/wiki/MUD) have done a lot of work on providing accessible terminal interfaces, but that hasn't filtered back to the various terminal apps.

@mcsf: As this is a highly visual issue, perhaps an appropriate regression test would be a bot that takes a screenshot of the G and posts it to every PR, for manual confirmation?

**Comment by noisysocks** [comment]
screen shot 2018-04-26 at 14 54 43

**Comment by jasmussen** [comment]
Did you address Riads' concerns? https://github.com/WordPress/gutenberg/pull/6418#pullrequestreview-115195151

**Comment by pento** [comment]
@jasmussen: I feel that Kanye sufficiently addressed @youknowriad's concerns. 

That said, I do understand how it may not have been entirely satisfactory. Thankfully, Kanye has chosen to re-address those concerns.

https://twitter.com/kanyewest/status/989268464392011776

**Comment by jasmussen** [comment]
Okay. I also see that you properly milestoned it too. We'll let it slide. This time. 

I'm watching you: 👈✌️👀

**Pull request merged**